capplets/common/mate-theme-apply.c

```diff
@@ -48,13 +50,26 @@ void
 mate_meta_theme_set (MateThemeMetaInfo *meta_theme_info)
 {
   GSettings *interface_settings;
+  GSettings *interface_gnome_settings;
   GSettings *marco_settings;
   GSettings *mouse_settings;
   GSettings *notification_settings = NULL;
   gchar *old_key;
   gint old_key_int;
+  gboolean wayland;
+
+  /*Find out if we are running under wayland and have xwayland
+   *Sending signal 0 with killall doesn't kill the process but
+   *the return value tells us whether the process exists
+   */
+  if (GDK_IS_X11_DISPLAY ((gdk_display_get_default())))
+    wayland = FALSE;
+
```

Comment:
Extra spaces

Reply:
This file was converted some time ago to GNOME style 2 space indents. That is the indent under the `if` statement,  same under the `else` statement. If this makes the file hard to read, I can use brackets instead if you or anyone else prefers.